packages/file-search/src/browser/quick-editors-open.ts

```diff
@@ -0,0 +1,176 @@
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { inject, injectable } from 'inversify';
+import {
+    QuickOpenModel, QuickOpenItem, QuickOpenMode, PrefixQuickOpenService,
+    OpenerService, QuickOpenGroupItem, QuickOpenGroupItemOptions, QuickOpenItemOptions,
+    QuickOpenHandler, QuickOpenOptions
+} from '@theia/core/lib/browser';
+import { ApplicationShell } from '@theia/core/lib/browser/shell/application-shell';
+import { FileSystem } from '@theia/filesystem/lib/common/filesystem';
+import { WorkspaceService } from '@theia/workspace/lib/browser/workspace-service';
+import { MessageService } from '@theia/core/lib/common/message-service';
+import URI from '@theia/core/lib/common/uri';
+import { LabelProvider } from '@theia/core/lib/browser/label-provider';
+import { Command } from '@theia/core/lib/common';
+import * as fuzzy from 'fuzzy';
+import { CancellationTokenSource } from '@theia/core/lib/common';
+import { EditorWidget } from '@theia/editor/lib/browser';
+
+export const SHOW_ALL_OPENED_EDITORS: Command = {
+    id: 'workbench.action.showAllEditors',
+    category: 'View',
+    label: 'Show All Opened Editors'
+};
+
+@injectable()
+export class QuickEditorsOpenService implements QuickOpenModel, QuickOpenHandler {
+
+    @inject(ApplicationShell)
+    protected readonly applicationShell: ApplicationShell;
+
+    @inject(FileSystem)
+    protected readonly fileSystem: FileSystem;
+
+    @inject(WorkspaceService)
+    protected readonly workspaceService: WorkspaceService;
+
+    @inject(OpenerService)
+    protected readonly openerService: OpenerService;
+
+    @inject(PrefixQuickOpenService)
+    protected readonly prefixQuickOpenService: PrefixQuickOpenService;
+
+    @inject(MessageService)
+    protected readonly messageService: MessageService;
+
+    @inject(LabelProvider)
+    protected readonly labelProvider: LabelProvider;
+
+    readonly prefix: string = 'edt ';
+
+    get description(): string {
+        return 'Show All Opened Editors';
+    }
+
+    getModel(): QuickOpenModel {
+        return this;
+    }
+
+    getOptions(): QuickOpenOptions {
+        return {
+            placeholder: '',
+            fuzzyMatchLabel: {
+                enableSeparateSubstringMatching: true
+            },
+            fuzzyMatchDescription: {
+                enableSeparateSubstringMatching: true
+            },
+            showItemsWithoutHighlight: true,
```

Comment:
We shouldn't have this option for the `show all opened editors`.
This means that even with fuzzy searching, items which are not matched will still appear.